cmd/finch/main.go

```diff
@@ -108,6 +110,7 @@ func virtualMachineCommands(
 		config.NewNerdctlApplier(fssh.NewDialer(), fs, fp.LimaSSHPrivateKeyPath(), system.NewStdLib()),
 		fp,
 		fs,
+		disk.NewUserDataDiskManager(lcc, &afero.OsFs{}, fp, system.NewStdLib().Env("HOME")),
```

Comment:
nit: any reason to not just pass `fs` instead of a new `afero.OsFs{}`?

Reply:
Because the persistent disk code needs to manage symlinks, but the generic `Fs` interface doesn't provide that. I made the `diskFS` interface in `pkg/disk/disk.go` for this purpose, but most other parts of finch can't use that because we want to be able to unit test them without interacting with the filesystem so we use `MemMapFs` which does not have symlinks implemented.

I could export the `diskFS` interface and pass that from main instead of an `Fs` but since nowhere else in finch currently needs it I felt it made the most sense to just instantiate a new `OsFs` here (especially since it doesn't store any state)

Reply:
Makes sense, thanks for the explanation!